editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -103,6 +105,21 @@ function Component(props: Props) {
     }
   );
 
+  // if allowNewAddresses is on, fetch the boundary geojson for this team to position the map view or default to London
+  //   example value for team.settings.boundary is https://www.planning.data.gov.uk/entity/8600093.geojson
+  const { data: geojson } = useSWR(
+    () =>
+      props.allowNewAddresses && team?.settings?.boundary
+        ? team.settings?.boundary
+        : null,
+    fetcher,
+    {
+      shouldRetryOnError: true,
+      errorRetryInterval: 500,
+      errorRetryCount: 1,
+    }
+  );
```

Comment:
Super simple ✅ 

I've added a comment to this Trello ticket to also populate `team.settings.bounday` whilst we're in the business of updating this table - https://trello.com/c/mDxLZdjO/2216-link-council-logo-at-top-of-planx-pages-to-council-website-homepages